gitter/messages.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import requests
+
+from data.webservices import webservices_url
+from gitter.score import sentiment_score
+from gitter.models import Question, Answer
+
+
+def get_messages():
+    """
+    Get all the messages send by newcomers on the gitter rooms.
+    """
+    logger = logging.getLogger(__name__)
+    IMPORT_URL = webservices_url('messages')
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=IMPORT_URL,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
+
+    data = response.json()
+    return data
+
+
+def message_score(message):
```

Comment:
there is no message scoring in https://github.com/coala/cEPs/blob/master/cEP-0020.md#gitter , so this method name is not appropriate.

You can use `score` terminology in the `gitter/nlp` namespace, but in the main app code there is only 'question', 'answer', and 'not known'.